packages/styling/components/button.scss

```diff
@@ -133,12 +261,36 @@ dso-label[status="danger"] {
 
 .btn {
   @include button-default-base();
+  @include button-size($padding-base-vertical, $padding-base-horizontal, $dso-button-font-size, $line-height-base, $dso-button-border-radius);
+
+  background-image: none; // Reset unusual Firefox-on-Android default style; see https://github.com/necolas/normalize.css/issues/214
+  border: $dso-button-border-width solid transparent;
+  cursor: pointer;
+  display: inline-block;
+  margin-bottom: 0; // For input.btn
+  text-align: center;
+  touch-action: manipulation;
+  vertical-align: middle;
+  white-space: nowrap;
+
+  &:hover,
+  &:focus,
+  &.focus {
+    color: $btn-default-color;
+    text-decoration: none;
+  }
 
   &:active,
-  &.active,
-  &:active:hover,
-  &.active:hover {
-    box-shadow: none;
+  &.active {
+    background-image: none;
+    outline: 0;
+  }
+
+  &.disabled,
+  &[disabled],
+  fieldset[disabled] & {
+    cursor: not-allowed;
+    opacity: .65;
   }
```

Comment:
Ook in een var?

Reply:
Is dat echt nodig? Gaat ie ooit anders worden?

Reply:
Is het geen algemene waarde die vaker gebruikt wordt? Of wordt `.65` alleen hier gebruikt?

Reply:
Nee hoor, die komt alleen hier voor